homeassistant/components/switchbee/button.py

```diff
@@ -0,0 +1,53 @@
+"""Support for SwitchBee scenario button."""
+import logging
+
+from switchbee.api import SwitchBeeError
+from switchbee.device import ApiStateCommand, DeviceType, SwitchBeeBaseDevice
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import SwitchBeeCoordinator
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Switchbee button."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
```

Comment:
Please add the type hint to the coordinator:
```suggestion
    coordinator: SwitchBeeCoordinator = hass.data[DOMAIN][entry.entry_id]
```